packages/physics-physx/src/PhysXPhysicsManager.ts

```diff
@@ -0,0 +1,230 @@
+import { PhysXPhysics } from "./PhysXPhysics";
+import { Ray, Vector3 } from "@oasis-engine/math";
+import { IPhysicsManager } from "@oasis-engine/design";
+import { PhysXCollider } from "./PhysXCollider";
+
+/** Filtering flags for scene queries. */
+export enum QueryFlag {
+  STATIC = 1 << 0,
+  DYNAMIC = 1 << 1,
+  ANY_HIT = 1 << 4,
+  NO_BLOCK = 1 << 5
+}
+
+/**
+ * Physics state
+ */
+export enum PhysicsState {
+  TOUCH_FOUND,
+  TOUCH_PERSISTS,
+  TOUCH_LOST,
+  TOUCH_NONE
+}
+
+/**
+ * A manager is a collection of bodies and constraints which can interact.
+ */
+export class PhysXPhysicsManager implements IPhysicsManager {
+  private static _tempPosition: Vector3 = new Vector3();
+  private static _tempNormal: Vector3 = new Vector3();
+  private static _pxRaycastHit: any;
+  private static _pxFilterData: any;
+
+  private _pxScene: any;
+
+  private readonly _onContactBegin?: Function;
+  private readonly _onContactEnd?: Function;
+  private readonly _onContactPersist?: Function;
+  private readonly _onTriggerBegin?: Function;
+  private readonly _onTriggerEnd?: Function;
+  private readonly _onTriggerPersist?: Function;
```

Comment:
Write Type with `: (obj1: number, obj2: number) => void` is more clear.

Reply:
done